XamlAnimatedGif.Shared/AnimationCompletedEventArgs.cs

```diff
@@ -0,0 +1,30 @@
+﻿#if WPF || SILVERLIGHT
+using System.Windows;
+#elif WINRT
+using Windows.UI.Xaml;
+#endif
+
+namespace XamlAnimatedGif
+{
+#if WPF
+    public delegate void AnimationCompletedEventHandler(DependencyObject d, AnimationCompletedEventArgs e);
+
+    public class AnimationCompletedEventArgs : RoutedEventArgs
+    {
+        public AnimationCompletedEventArgs(object source)
+            : base(AnimationBehavior.AnimationCompletedEvent, source)
+        {
+#elif WINRT || SILVERLIGHT
+    public class AnimationCompletedEventArgs : EventArgs
```

Comment:
This doesn't build because of a missing `using System;`. Since the namespace is necessary only for WinRT and Silverlight, it's probably easier to just write the fully qualified name.

Reply:
@thomaslevesque , changed EventArgs => System.EventArgs